pages/common/nmap.md

```diff
@@ -4,6 +4,20 @@
 > Some features only activate when Nmap is run with privileges.
 > More information: <https://nmap.org>.
 
+# nmap
+> network scanning tool
+> More information: <https://nmap.org/>.
+
+- If Nmap throws a root privileges error use it with `sudo` or `admin rights cmd`
```

Comment:
We don't include this kind of message in our pages.

Reply:
Generally our format is to have a description, followed by an example.

Also, we try to keep pages focused on the command in question, and avoid explaining generic shell concepts such as `sudo`. We do have a convention to prefix commands that require sudo with `sudo` in the example, but in this case as far as I know `nmap` doesn't require `sudo` to function (it just takes longer).

To that end, I'd suggest we remove this line.

Reply:
Done!